df_py/predictoor/csvs.py

```diff
@@ -1,5 +1,8 @@
 import csv
 import os
+import glob
+import re
+
```

Comment:
(Commenting here because it's hard to insert comments into files that currently have no changes)

In df_py/volume/csvs.py, we should rename the functions to (a) make the "volume" part explicit just as "predictoor" and "challenge" is explicit, and (b) ensure snake case. That is:
```
saveRewardsperlpCsv --> save_volume_rewards_csv
loadRewardsCsv --> load_volume_rewards_csv
rewardsperlpCsvFilename --> volume_rewards_csv_filename

saveRewardsinfoCsv --> save_volume_rewardsinfo_csv
rewardsinfoCsvFilename --> volume_rewardsinfo_csv_filename
```

----

While you're at it, make the other functions in df_py/volume/csvs.py snake case too.

----

While you're at it, make the functions in df_py/challenge/csvs.py snake case too.